src/views/LeadAdd/LeadDetail/index.js

```diff
@@ -75,6 +77,26 @@ const AuthorEmptyComponent = () => (
 const FaramBasicSelectInput = FaramInputElement(BasicSelectInput);
 const ModalButton = Modalize(Button);
 
+const capitalize = string => (
+    string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+);
+
+function isUrlValid(url) {
+    return (requiredCondition(url).ok && urlCondition(url).ok);
+}
+
+const getTitleFromUrl = (url) => {
+    if (!isUrlValid(url)) {
+        return undefined;
+    }
+    let title = url.match(/\/([^/?]+)(?:\?.*)?$/)[1];
+    title = title.replace(/(\.\w{1,5})+$/, '');
+    title = title.replace(/_/g, ' ');
+    title = title.replace(/-/g, ' ');
```

Comment:
Both of these can be done with a single regex.
Plus, I think you want to use replaceAll here